code/src/main/webapp/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
Maybe create separate folders for html, css and js. I think Grady said he wanted the project to be structured better than the portfolio.